Library/OcAppleEventLib/PointerHandler.c

```diff
@@ -725,6 +752,11 @@ InternalSimplePointerPollNotifyFunction (
     CommonStatus = EFI_NOT_READY;
 
     for (Index = 0; Index < mNumberOfPointerProtocols; ++Index) {
+      if (mSimplePointerPollMask != POINTER_POLL_ALL_MASK
+        && (LShiftU64 (1, Index) & mSimplePointerPollMask) == 0) {
```

Comment:
Why U64 when we are limited to U32 by the PLIST format (and mSimplePointerPollMask in fact is UINT32 already)? Also, should we maybe warn if there are pointer protocols that cannot be described by the mask? Will this not be UB for Index >= 64?